skore/src/skore/item/cross_validation_aggregation_item.py

```diff
@@ -36,10 +37,19 @@ def plot_cross_validation_aggregation(
 
     _cv_results = cv_results_items_versions.copy()
 
-    df = pandas.DataFrame([v.cv_results_serialized for v in _cv_results])
-    df = df.drop(columns=["indices", "estimator"], errors="ignore")
-    df = df.apply(pandas.Series.explode)
-    df = df.reset_index(names="run_number")
+    cv_list = [v.cv_results_serialized for v in _cv_results]
+    df_list = []
+    for run, cv in enumerate(cv_list):
+        with contextlib.suppress(KeyError):
+            del cv["indices"]
+            del cv["estimator"]
```

Comment:
Can `estimator` be present if `indices` isn't? If so, you will never reach the line 45.

Reply:
That's fair, I'll do a loop then!